biorbd_optim/simulate.py

```diff
@@ -0,0 +1,66 @@
+import numpy as np
+
+
+class Simulate:
+    @staticmethod
+    def from_solve(ocp, sol):
+        v = np.array(sol["x"]).squeeze()
+        offset = 0
+        for nlp in ocp.nlp:
+            # TODO adds StateTransitionFunctions between phases
+            for idx_nodes in range(nlp["ns"]):
+                v[offset + nlp["nx"] + nlp["nu"] : offset + 2 * nlp["nx"] + nlp["nu"]] = np.array(
+                    nlp["dynamics"][idx_nodes](
+                        x0=v[offset : offset + nlp["nx"]], p=v[offset + nlp["nx"] : offset + nlp["nx"] + nlp["nu"]]
+                    )["xf"]
+                ).squeeze()
```

Comment:
If I am not mistaken, this performs multi_shoot simulation (which is fine and I think should be the default option here), but it should also allow for `single_shoot=True`